relay-server/src/services/health_check.rs

```diff
@@ -98,17 +113,37 @@ impl HealthCheckService {
                     return false;
                 }
 
-                self.project_cache
-                    .send(SpoolHealth)
+                let project_cache_sent = Instant::now();
+                let result = self
+                    .project_cache
+                    .send(SpoolHealthWrapper {
+                        message: SpoolHealth,
+                        sent_instant: project_cache_sent,
+                    })
                     .await
-                    .unwrap_or_default()
+                    .unwrap_or_default();
+                relay_statsd::metric!(
+                    timer(RelayTimers::HealthcheckServiceProjectcacheResponseDuration) =
```

Comment:
I think we should have one metric for health check probes and tag it with what operation, e.g. `project_cache`, meaning we can instrument all of our probes easily.